index.html

```diff
@@ -0,0 +1,177 @@
+          </div>
+          <h3 class="program-title">Lecture</h3>
+          <p class="p-1">
+            Listen to the snooker lessons taught by the most-experienced
+            international grandmasters.
+          </p>
+        </section>
+        <section class="program-card col-md-2">
+          <div class="icon-container">
+            <i class="fas fa-chess-board"></i>
+          </div>
+          <h3 class="program-title">Match</h3>
+          <p class="p-1">
+            Watch the exciting snooker matches by the top players and learn from
+            them.
+          </p>
+        </section>
+        <section class="program-card col-md-2">
+          <div class="icon-container">
+            <i class="fas fa-chess-queen"></i>
+          </div>
+          <h3 class="program-title">Practice</h3>
+          <p class="p-1">
+            Play snooker with other players over a chessboard, rather than
+            playing online.
+          </p>
+        </section>
+        <section class="program-card col-md-2">
+          <div class="icon-container">
+            <i class="fas fa-book"></i>
+          </div>
+          <h3 class="program-title">Theory</h3>
+          <p class="p-1">
+            Study the important theory about the openings, middle game and
+            endgame.
+          </p>
+        </section>
+        <section class="program-card col-md-2">
+          <div class="icon-container">
+            <i class="fas fa-users"></i>
+          </div>
+          <h3 class="program-title">Connect</h3>
+          <p class="p-1">
+            Become part of the amazing snooker community to discuss, study and
+            play.
+          </p>
+        </section>
+      </section>
+      <button type="button" id="join-btn">Join Event</button>
+    </article>
+    <article class="feature-container d-flex flex-column align-items-center p-3 bg-white" id="player-section">
+      <h2 class="section-heading">Featured Players</h2>
+      <hr />
+      <section class="row justify-content-center">
+        <section class="col-md-5 d-flex align-items-center p-1 speaker" id="player1">
+          
+        </section>
+        <section class="col-md-5 d-flex align-items-center p-1 speaker" id="player2">
+          
+        </section>
+        <section class="col-md-5 d-flex align-items-center p-1 speaker" id="player3">
+   
+        </section>
+        <section class="col-md-5 d-flex align-items-center p-1 speaker" id="player4">
+          
+        </section>
+        <section class="col-md-5 d-flex align-items-center p-1 speaker" id="player5">
+          
+        </section>
+        <section class="col-md-5 d-flex align-items-center p-1 speaker" id="player6">
+          
+        </section>
+        <section class="col-md-5 d-flex align-items-center p-1 speaker" id="player7">
+          
+        </section>
+        <section class="col-md-5 d-flex align-items-center p-1 speaker" id="player8">
```

Comment:
- [ ] The more button in the speaker section should show all the speakers that are not currently displayed on loading the page. Kindly implement this.
-
![image](https://user-images.githubusercontent.com/81932478/204019155-2e8fea38-0f1d-4304-8b27-6eb644b8fcfa.png)